test/make.sh

```diff
@@ -48,8 +48,10 @@ function docker() {
 # files ending in '.tf'
 function check_terraform() {
   echo "Running terraform validate"
-  #shellcheck disable=SC2156
-  find . -name "*.tf" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
+  REPO_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+  cd "${REPO_ROOT}/terraform" || exit
+  terraform init
+  terraform validate
```

Comment:
we are gonna need to do this across all the projects, can you ping Kristen and backlog this please?